client/components/Dashboard/index.jsx

```diff
@@ -71,93 +71,68 @@ const QuickStartGuide = () => {
       <Header.Subheader className="dashboard-subheader">
         Learn to author and use Teacher Moments scenarios.
       </Header.Subheader>
-      <List ordered className="dashboard-grid dashboard-grid--numbered">
+      <List ordered className="dashboard-list">
         <List.Item>
-          <Card className="dashboard-card">
-            <p className="dashboard-card__title">What Is Teacher Moments?</p>
-            <p>An overview of Teacher Moments and ELK</p>
-            <a
-              className="dashboard-card__link"
-              href="https://docs.google.com/presentation/d/1671Hfy4dIOjfPRrEr_-MWh_XRB7uZ2ACPiS8v1HWng4/edit#slide=id.gf16183f65e_0_54"
-            >
-              Go to guide →
-            </a>
-          </Card>
+          <a
+            className="dashboard-list__title"
+            href="https://docs.google.com/presentation/d/1671Hfy4dIOjfPRrEr_-MWh_XRB7uZ2ACPiS8v1HWng4/edit#slide=id.gf16183f65e_0_54"
+          >
+            What Is Teacher Moments?
+          </a>
+          <p>An overview of Teacher Moments and ELK</p>
         </List.Item>
         <List.Item>
-          <Card className="dashboard-card">
-            <p className="dashboard-card__title">Authoring a Scenario</p>
-            <p>How to develop a basic scenario in Teacher Moments</p>
-            <a
-              className="dashboard-card__link"
-              href="https://docs.google.com/presentation/d/1GgTQMwtcYWYHzWjzQYdEtN3RiUCn8aF7NPn-6Eb7N6c/edit#slide=id.p"
-            >
-              Go to guide →
-            </a>
-          </Card>
+          <a
+            className="dashboard-list__title"
+            href="https://docs.google.com/presentation/d/1GgTQMwtcYWYHzWjzQYdEtN3RiUCn8aF7NPn-6Eb7N6c/edit#slide=id.p"
+          >
+            Authoring a Scenario
+          </a>
```

Comment:
@rwaldron good q! Generally I like to avoid opening up in new tabs [for accessibility reasons](https://codersblock.com/blog/external-links-new-tabs-and-accessibility/). So I think we can keep these as is for now.